tests/integration/kubernetes/run_kubernetes_nv_tests.sh

```diff
@@ -11,6 +11,16 @@ kubernetes_dir=$(dirname "$(readlink -f "$0")")
 # shellcheck disable=SC1091 # import based on variable
 source "${kubernetes_dir}/../../common.bash"
 
+# Enable NVRC trace logging for NVIDIA GPU runtime
+enable_nvrc_trace() {
+	if [[ ${RUNTIME_CLASS_NAME:-kata-qemu-nvidia-gpu} == "kata-qemu-nvidia-gpu" ]]; then
+		config_file="/opt/kata/share/defaults/kata-containers/configuration-qemu-nvidia-gpu.toml"
+	fi
+	if ! grep -q "nvrc.log=trace" "${config_file}"; then
+		sudo sed -i -e 's/^kernel_params = "\(.*\)"/kernel_params = "\1 nvrc.log=trace"/g' "${config_file}"
+	fi
+}
+
 cleanup() {
```

Comment:
Why move this here to this file?

Reply:
I wanted to move this function outside of individual bats files - likely we'll have more bats files being added. I was unsure whether `common.bash` or `tests_common.sh` would be suitable or not (would maybe become more suitable if we'd author a generic function which would append a string to the kernel command line, but unsure whether we want to do this or not.